android/src/ch/opengis/qfield/QFieldProjectActivity.java

```diff
@@ -253,6 +296,21 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         Log.d(TAG, "onActivityResult ");
         Log.d(TAG, "resultCode: " + resultCode);
 
+        Log.d(TAG, "data: " + data.getDataString());
+        Log.d(TAG, "requestCode: " + requestCode);        
+
+        // This is the result of ACTION_OPEN_DOCUMENT_TREE 
+        if(requestCode == 666){
```

Comment:
:imp:

Reply:
seriously!? they really don't want people to use files...

Reply:
It's a code I chose. But it seemed to me to be an appropriate number.